### PR TITLE
Error correction

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
- changed names
 - added new validation for versionSpec
 - update README
 - add yml files

### CODE REVIEW

File: src/find-pypy.ts [hunk 1]
```diff
@@ -114,3 +121,9 @@ function parsePyPyVersion(versionSpec: string): IPyPyVersionSpec {
 function getPyPyVersionFromPath(installDir: string) {
   return path.basename(path.dirname(installDir));
 }
+
+function validatePythonVersion(version: string) {
```
Comment (maxim-lobanov) [inline src/find-pypy.ts:125]: validateVersion

File: src/find-pypy.ts [hunk 2]
```diff
@@ -93,10 +93,17 @@ function parsePyPyVersion(versionSpec: string): IPyPyVersionSpec {
...
-    throw new Error(
-      "Invalid 'version' property for PyPy. PyPy version should be specified as 'pypy-<python-version>'. See readme for more examples."
+    core.setFailed(
+      "Invalid 'version' property for PyPy. PyPy version should be specified as 'pypy-<python-version>'. See README for examples and documentation."
     );
   }
+
+  if (!validatePythonVersion(versions[1])) {
+    core.setFailed(
+      "Invalid 'version' property for PyPy. PyPy version should be specified as 'pypy-<python-version>'. See README for examples and documentation."
```
Comment (maxim-lobanov) [inline src/find-pypy.ts:103]: PyPy version should be specified as 'pypy-<python-version>' -> 
```
Both Python version and PyPy versions should satisfy SemVer notation.
```

File: src/find-pypy.ts (hunk 1 above)
Comment (dmitry-shibanov) [reply src/find-pypy.ts]: Done.

File: src/find-pypy.ts (hunk 2 above)
Comment (dmitry-shibanov) [reply src/find-pypy.ts]: Done.

**Pull request merged**